### DISCUSSION TITLE
Как пользоваться этим приложением?

### DISCUSSION THREAD

**Discussion by regulator1992** [Chimildic/YaMuTools#42] [Вопрос/ответ] [answered]
Установил, но при нажатии на иконку выдает страницу с настройками. А на самом сайте яндекс музыки ничего не изменилось.

**Comment by Chimildic** [comment]
Кнопки расширения появляются на отдельной  странице плейлиста (не боковое) и в общем списке плейлистов.

**Comment by Chimildic** [comment, marked as answer]
С версии 0.8.0 при первой установке появляется краткая справка о том как пользоваться расширением.
